packages/confirmation-modal/src/ConfirmationModal/ConfirmationModal.tsx

```diff
@@ -59,6 +59,7 @@ export const ConfirmationModal = React.forwardRef(
             // eslint-disable-next-line jsx-a11y/no-autofocus
             autoFocus
             darkMode={darkMode}
+            data-testid="lg-confirmation-modal-confirmation-input"
```

Comment:
Is this the pattern we will use in all our components, `lg-[component-name]`?

Reply:
Do we have a different pattern elsewhere?

Reply:
I remember @TheSonOfThomp having a proposal about naming our test IDs, but that was a while ago. I guess if it's the same everywhere else, then it's fine.